ssl/ssl_cert.c

```diff
@@ -801,8 +800,13 @@ STACK_OF(X509_NAME) *SSL_load_client_CA_file_ex(const char *file,
     goto done;
 
  file_err:
-    ret = NULL;
-    return ret;
+    if (in != NULL) {
+        BIO_free(in);
+    }
+    if (name_hash != NULL) {
+        lh_X509_NAME_free(name_hash);
+    }
```

Comment:
Please remove the extraneous ifs. All free functions are supposed to handle NULL argument gracefully.

Reply:
Sure, I can remove that. Should I also remove it in the other files?